README.md

```diff
@@ -1,8 +1,376 @@
+eudico                      latest        8fb6db609712   2 minutes ago   341MB
+```
+
 ## Usage
-- Build
-- Config
+### Configuration
+If you are running the agent for the first time, the first thing you need to do is to create a new config. The default config path for the agent is `~/.ipc-agent/config.toml`. The agent will always try to pick up the config from this path unless told otherwise. To populate a sample config file in the default path, you can run the following command:
+```bash
+$ ./bin/ipc-agent init-config
+```
+If you `cat ~/.ipc-agent/config.toml` you should see a new config populated with a sample root and subnet configurations.
+
+### Running the daemon
+The IPC agent runs as a foreground daemon process that spawns a new JSON RPC server to interact with it, and all the processes to automatically handle checkpoints and the execution of cross-net messages for the subnets our agent is participating in. The agent determines the list of subnets it should interact with from its config file. 
+
+Alternatively, the agent can also be used as a CLI to interact with IPC. Under the hood, this cli sends new commands to the RPC server of the daemon. To run the IPC agent daemon you can run:
+```
+$ ./bin/ipc-agent daemon
+```
+
+Running the agent at this point will throw an error, because we haven´t configured it to interact with any IPC network. In the next few sections we will walk you through different alternatives to spawn and connect your agent to a running IPC instance.
+
+The RPC server of the daemon will be listening to the endpoint determined in the `json_rpc_address` field of the config. If you are looking for your agent to be accessible from Docker or externally, remember to listen from `0.0.0.0` instead of `127.0.0.1` as specified in the empty configuration. 
+
+
+### Interacting with a rootnet
+#### Spacenet
+For more information about the Spacenet testnet have a look at the [Spacenet repo](https://github.com/consensus-shipyard/spacenet). In this section we will guide you through how to connect your IPC agent to a running instance of Spacenet. Spacenet hosts all the IPC actors and can be used as a rootnet to deploy new subnets from. 
+
+In order to use the IPC agent with Spacenet we need to have access to a full-node syncing with the network. The easiest way to achieve this is to run your own Spacenet node. Running you own Spacenet node is as simple as cloning Eudico repo,compiling and running it:
+```bash
+$ git clone https://github.com/consensus-shipyard/lotus
+
+# `spacenet` is the master branch of the repo. To find the latest
+# release deployed over spacenet, you can check the last release 
+# published in the repo and use checkout that tag.
+$ git checkout <release/branch>
+
+# Compile eudico for Spacenet
+$ make spacenet
+
+# Run your node
+$ ./eudico mir daemon --bootstrap=true
+```
+With this, your node should automatically connect to the bootstraps of the network and start syncing the latest state of the chain.
```

Comment:
Also, we'll need snapshots sooner rather than later. I expected sync for such a young network to be way faster.

Reply:
It is super spammy, this is a reminiscent from buildernet where storage functionalities had work, and we required fake randomness because our block time was faster than DRAND's period. I will add a PR to remove this in a minute.